problems/sevenBoom.js

```diff
@@ -4,8 +4,23 @@
 * @param {number} n - The number to count up to
 * @returns {number[]} - An array matching the pattern described above
 */
-function sevenBoom() {
+const sevenBoom = (n) => {
+let output;
+for(let i = 1; i <= 20; i++){
+    if(i % 7 === 0 || i % 10 ===7) {
```

Comment:
These conditions won't always work, what if the number was 73. We need to turn the num into a string and check if it includes "7"